packages/tasks/src/model-libraries.ts

```diff
@@ -214,6 +214,15 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 			term: { path: "gliner_config.json" },
 		},
 	},
+	glyph-byt5: {
```

Comment:
Some repos will need to have `library_name: glyph-byt5` before we merge this

Reply:
Hi, thanks for the review, I have added library name in https://huggingface.co/GlyphByT5/Glyph-SDXL-v2/tree/main